.github/workflows/tf_validate_ver.yml

```diff
@@ -0,0 +1,73 @@
+---
+name: Validate examples and modules against variety of TF versions
+
+on:
+  # push:
+  #   branch:
+  #     - develop
```

Comment:
Do we enable it?

Reply:
at some point - yes, initially the idea is to run it manually and then merge it into the 'regular' ci/cd workflow